docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -47,6 +47,102 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+## Select the edge of a used range
+
+An add-in can select the edge of the used range, based on the currently selected range. A used range is any cell or series of contiguous cells that have been edited in the worksheet. In the following screenshot, the used range is the table with values in each cell, **C5:F12**. The empty cells outside this table are outside the used range.
+
+![A table with data from C5:F12 in Excel.](../images/excel-ranges-used-range.png)
+
+### Select the cell at the edge of the current used range (online-only)
+
+> [!NOTE]
+> The `Range.getRangeEdge` method is currently only available in ExcelApiOnline 1.1. To learn more, see [Excel JavaScript API online-only requirement set](../reference/requirement-sets/excel-api-online-requirement-set.md).
+
+The following code sample shows how use the [Range.getRangeEdge](/javascript/api/excel/excel.range#getRangeEdge_direction__activeCell_) method to select the cell at the furthest edge of the current used range, in the direction up. This action matches the result of using the Ctrl+Arrow key keyboard shortcut while a range is selected.
+
+```js
+Excel.run(function (context) {
+    // Get the selected range.
+    var range = context.workbook.getSelectedRange();
+
+    // Specify the direction with the `KeyboardDirection` enum.
+    var direction = Excel.KeyboardDirection.up;
+
+    // Get the active cell in the workbook.
+    var activeCell = context.workbook.getActiveCell();
+
+    // Get the top-most cell of the current used range.
+    // This method acts like the Ctrl+Arrow key keyboard shortcut while a range is selected.
+    var rangeEdge = range.getRangeEdge(
+      direction,
+      activeCell
+    );
+    rangeEdge.select();
+
+    await context.sync();
+}).catch(errorHandlerFunction);
+```
+
+> [!NOTE]
+> If the selected range is outside a used range, the `Range.getRangeEdge` method will select the cell at the furthest edge of the worksheet in the chosen direction.
+
+#### Before selecting the cell at the edge of the used range
+
+The following screenshot shows a used range and a selected range within the used range. The used range is a table with data at **C5:F12**. Inside this table, the range **D8:E9** is selected. This selection is the *before* state, prior to running the `Range.getRangeEdge` method.
```

Comment:
I'm on the fence about including these screenshot explanations. I think they do a good job explaining the action, but increase the separation of the images from the code.

Reply:
I think that, unless you're thoroughly familiar with the UI experience, this concept is so confusing that the screenshots provide a helpful frame of reference. 

Now that the online-only notes are moved, does the content flow feel less broken up?

Reply:
Agreed that the screenshots are necessary. I was referring to the text blocks (apologies for not being clear):

![image](https://user-images.githubusercontent.com/38896772/122999153-c036ba80-d362-11eb-9b63-538085fcf480.png)

Reply:
Oh, I see! I included the descriptions explicitly for accessibility, in case a person can't see the screenshot. They're outside the alt text because it's too long for alt text, and it might benefit people who *can* see the screenshot. I'll consult our editor for an additional perspective.

Reply:
Way to be inclusive! Yes, I agree the text describing the following screenshot is necessary for accessibility.